src/Cli/dotnet/commands/dotnet-msbuild/MSBuildLogger.cs

```diff
@@ -138,6 +141,21 @@ internal static void FormatAndSend(ITelemetry telemetry, TelemetryEventArgs args
                         toBeHashed: Array.Empty<string>(),
                         toBeMeasured: new[] { "FileLoggersCount" });
                     break;
+                case BuildcheckAcquisitionFailureEventName:
+                    TrackEvent(telemetry, $"msbuild/{BuildcheckAcquisitionFailureEventName}", args.Properties,
+                        toBeHashed: new[] { "AssemblyName", "ExceptionType", "ExceptionMessage" },
+                        toBeMeasured: Array.Empty<string>());
+                    break;
+                case BuildcheckRunEventName:
+                    TrackEvent(telemetry, $"msbuild/{BuildcheckRunEventName}", args.Properties,
+                        toBeHashed: Array.Empty<string>(),
+                        toBeMeasured: new[] { "RulesCount", "CustomRulesCount", "ViolationsCount", "TotalRuntimeInMilliseconds" });
```

Comment:
Basically the only thing that should be a 'measure' is a time duration. Everything else should be a normal property. Measures get min/max/stddev calculations done on them in a way that isn't helpful and just clutters up the events/GDPR tools.

Reply:
Aaah - good to know!